plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/logging/RemoteQueryModifierModule.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc.logging;
+
+import com.google.inject.Binder;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+
+import static com.google.inject.Scopes.SINGLETON;
+import static io.airlift.configuration.ConditionalModule.conditionalModule;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+
+public class RemoteQueryModifierModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        configBinder(binder).bindConfig(RemoteQueryModifierConfig.class);
+        install(conditionalModule(
+                RemoteQueryModifierConfig.class,
+                config -> !config.getFormat().isBlank(),
```

Comment:
If the pattern is blank we bind `RemoteQueryModifier.NONE` so instead of having it as blank - can we use `Optional` and maybe add an assertion in the config to ensure we don't inject blank values ?

Reply:
hm I guess you are right.